apps/cricketscoreapi/README.md

```
@@ -0,0 +1,99 @@
+2. Clerk will automatically handle the authentication tokens
+3. Make requests from your frontend application with the Clerk session
+
+## 🧪 Running Tests
+
+After setting up the project, you can run the tests to ensure everything is working as expected. Follow these steps:
+
+1. Run the test command: `pnpm test`
+
+This command will run the test suite for the project.
+
+2. If you want to check the code coverage of your tests, you can run: `pnpm test:coverage`
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)